ui/components/app/assets/nfts/nfts-tab/nfts-tab.tsx

```diff
@@ -119,21 +151,62 @@ export default function NftsTab() {
             <NFTsDetectionNoticeNFTsTab />
           </Box>
         ) : null}
-        {hasAnyNfts > 0 || previouslyOwnedCollection.nfts.length > 0 ? (
+        {hasAnyNfts || previouslyOwnedNfts.length > 0 ? (
           <Box>
-            <NftsItems
-              collections={collections}
-              previouslyOwnedCollection={previouslyOwnedCollection}
+            <NftGrid
+              nfts={sortedNfts}
+              handleNftClick={handleNftClick}
+              privacyMode={privacyMode}
```

Comment:
`NftGrid` now does what `NftItems` used to do. So, maybe it makes sense to just remove `NftItems` entirely in this PR.